### PR TITLE
Kieran Murphy laravel test

### PR DESCRIPTION
**Pull request by kmurp62rulz**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-MAIL_PASSWORD=null
-MAIL_ENCRYPTION=null
-
-PUSHER_APP_ID=
-PUSHER_APP_KEY=
-PUSHER_APP_SECRET=
-PUSHER_APP_CLUSTER=mt1
-
-MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
-MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
```
Comment (stevethomas) [inline .env.example:39]: no need to delete this file... it gives anyone starting on the project a starting point for .env; often with some useful development config / keys.

File: app/posts.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class posts extends Model
```
Comment (stevethomas) [inline app/posts.php:7]:
```suggestion
class Post extends Model
```

File: database/migrations/2021_02_05_023128_create_posts_table.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->increments('ID');
+            $table->integer('User ID');
```
Comment (stevethomas) [inline database/migrations/2021_02_05_023128_create_posts_table.php:18]:
```suggestion
            $table->increments('id');
            $table->integer('user_id');
```

File: database/migrations/2021_02_05_023239_posts.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<?php
```
Comment (stevethomas) [inline database/migrations/2021_02_05_023239_posts.php:1]: this migration is a duplicate of the other one?

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+            $table->timestamps();
+        });
+    }
+```
+
+
+## What I found hard
+I had many issues with this test as it was my first time working with a PHP based framework.
+I installed many things such as composer, Laravel, TablePlus, DBngin etc to be able to work on the test. 
+Unfortunately my main issues came from connecting to the local mySQL database to be able to make migrations to the database. I followed many tutorials, read many posts on stackoverflow.com and emailed with Jerrell unfortunately I wasn't able to resolve my errors.
```
Comment (stevethomas) [inline readme.md:25]: It is a big chunk of learning just to get a local dev environment up and running; well done for getting as far as you did. 

I'm sure the connection issue will turn out to be trivial, probably either MySQL is not running, or the credentials you are connecting with are incorrect or do not have access to the database you are trying to access.

Comment (kmurp62rulz) [reply readme.md:25]: Thank you for the feedback and taking the time to review my code. Really appreciate it! As you can tell its my first time dealing with Laravel but I hope I can see myself improve in the future.

**Pull request closed without merging**